### PR TITLE
:bug: install graphviz/dot

### PR DESCRIPTION
**Pull request by joerunde**
PR checks are having issues because:
- pydeps needs `dot` installed
- some weird bug with 1.12.5 crashes :(

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -37,7 +37,7 @@ commands = pylint caikit
 
 [testenv:imports]
 description = enforce internal import rules
-deps = pydeps
+deps = pydeps==1.12.3
```
Comment (gabe-l-hart) [inline tox.ini:40]: Version range?

**Review by gabe-l-hart — APPROVED** [review]
Looks good. Might be better with a version range?

File: tox.ini (hunk 1 above)
Comment (joerunde) [reply tox.ini:40]: maaaybe, but this thing literally released half an hour ago and bricked the builds lol. This version works and has the functionality we need so I'd kinda rather just stick to it for stability for this tiny lil check

**Comment by joerunde** [comment]
The check lives!

**Pull request merged**